README.md

```diff
@@ -0,0 +1,127 @@
+
+[![](https://jitpack.io/v/tillhub/scan-engine.svg)](https://jitpack.io/#tillhub/scan-engine)
+[![API](https://img.shields.io/badge/API-24%2B-green.svg?style=flat)](https://android-arsenal.com/api?level-11)
+# Scan Engine
+
+This library combines various scanning protocols into a single, easy-to-use interface.
+
+# How to setup
+
+**Step 1.** Add the JitPack repository to your `settings.gradle` file:
+
+```groovy
+dependencyResolutionManagement {
+    repositories {
+        ...
+        mavenCentral()
+		maven { url 'https://jitpack.io' }
+    }
+}
+```
+
+**Step 2.** Add the dependency to your app `build.gradle`:
+```groovy
+dependencies {
+    implementation 'com.github.tillhub:scan-engine:x.x.x'
+}
+```
+# Usage
+
+There are two ways you can use scan SDK:
+* `Camera scanner` : Manually register an activity or fragment result callback for the camera scanner.
+* `Barcode scanner` : Register a BroadcastReceiver with the barcode scanner.
+
+### 1. Camera Scanner (Activity/Fragment):
```

Comment:
I would remove the "Activity/Fragment" here, it is unnecessarily precise.
You could word it as "Spawns a separate Activty"